server/storage/backend/verify.go

```diff
@@ -46,6 +46,15 @@ func ValidateCalledOutSideApply(lg *zap.Logger) {
 	}
 }
 
+func ValidateCalledFromUnittest(lg *zap.Logger) {
```

Comment:
```suggestion
func ValidateCalledInsideUnittest(lg *zap.Logger) {
```
Let's be consistent in naming functions here.

Reply:
Either way is fine, however let's have a one way to express stacktrace analysis.

Reply:
done.